### DISCUSSION TITLE
Configurar Impresora 3D Hellbot Magna 1 V1. para Creality Sonic Pad

### DISCUSSION THREAD

**Discussion by MAIE3452** [jschuh/klipper-macros#236] [Q&A]
Hola a toda la comunidad, les consulto: para poder configurar la Magna 1, necesito dos archivos: 
Uno que delega los comandos a Klipper con extension ".bin" y Otro que se llama "printer.cfg"
Como no se como realizarlos y no los encuentro en la web, Pregunto si alguien los tiene me los puede Copartir? o bien me indiquen de donde descargarlos.
Desde ya, muchisimas gracias por la colaboracion.

**Comment by altamiranoaxel77** [comment]
pudiste solucionarlo?

**Comment by mircsicz** [comment]
Este no es el propio Repository para descargar el printer.cfg, posiblamente un printer.cfg para tu modello toda via no existe...
